package.json

```diff
@@ -37,7 +37,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#7522d68c6a61a5bf535672670ef6195e610f2fa0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#40fd505dcc0783c5af479642beaa566fbe5515ce",
```

Comment:
@ethervoid It should have different commit for master and release-5.26.
- master should be https://github.com/mattermost/mattermost-redux/commit/3f5b8566208b7b3116fb20b4ad8278ddbae2a0a3
- release-5.26 should be https://github.com/mattermost/mattermost-redux/commit/40fd505dcc0783c5af479642beaa566fbe5515ce